src/index.html

```diff
@@ -4,9 +4,298 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a href="#">
+            <img src="./images/logo.png" alt="moyo logo">
+          </a>
+        </div>
+        <nav class="navigation header__navigation">
+          <ul class="navigation__container">
+            <li class="navigation__item">
+              <a
+                class="navigation__link is-active"
+                href="#apple">Apple
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#samsung">Samsung</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#smartphones">Smartphones</a>
+            </li>
+            <li class="navigation__item">
+              <a
+                class="navigation__link"
+                href="#laptops_computers"
+                data-qa="nav-hover">Laptops & Computers
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#gadgets">Gadgets</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#tablets">Tablets</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#photo">Photo</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#video">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <div class="card__image-block">
+          <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Card image"
+        >
+        </div>
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          <div class="card__product-code">
+            Product code: 195434
+          </div>
```

Comment:
- Use semantic tags where possible
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/0c3e631b-cdeb-42e0-8307-71bdc193d3c0)
- And I have no idea why you put all other elements inside `card_title`, fix it